pom.xml

```diff
@@ -1,67 +1,73 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>com.devonfw</groupId>
-    <artifactId>maven-parent</artifactId>
-    <version>6</version>
-  </parent>
-  <groupId>com.devonfw.java.dev</groupId>
-  <artifactId>devon4j</artifactId>
-  <version>dev-SNAPSHOT</version> <!-- This POM will never be released -->
-  <packaging>pom</packaging>
-  <name>${project.artifactId}</name>
-  <description>Java stack of devonfw.com (devon4j): develop Java apps standardized, faster and with less risk.</description>
-  <inceptionYear>2014</inceptionYear>
-
-  <modules>
-    <module>boms</module>
-    <module>modules</module>
-    <module>starters</module>
-    <module>templates</module>
-    <module>documentation</module>
-  </modules>
-
-  <properties>
-    <revision>dev-SNAPSHOT</revision>
-    <github.repository>devon4j</github.repository>
-    <devon4j.version>${revision}</devon4j.version>
-    <spring.boot.version>2.4.4</spring.boot.version>
-    <!-- Spring boot and spring cloud version has to match -->
-    <spring.cloud.dependencies.version>2020.0.0</spring.cloud.dependencies.version>
-    <jackson.version>2.12.2</jackson.version> <!-- Overriding Jackson for fixing vulnerabilities -->
-    <guava.version>30.1.1-jre</guava.version>
-    <junit.version>5.7.1</junit.version>
-    <cxf.version>3.4.3</cxf.version>
-    <mmm.util.version>8.7.0</mmm.util.version>
-    <slf4j.version>1.7.30</slf4j.version>
-    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
-    <sonar.organization>devonfw</sonar.organization>
-  </properties>
-
-  <build>
-    <plugins>
-
-      <!-- avoid version in local war files, exclude development properties from WARs -->
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-war-plugin</artifactId>
-        <configuration>
-          <packagingExcludes>WEB-INF/classes/config/application.properties,*.jsp</packagingExcludes>
-          <warName>${project.artifactId}</warName>
-        </configuration>
-      </plugin>
-
-    </plugins>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.springframework.boot</groupId>
-          <artifactId>spring-boot-maven-plugin</artifactId>
-          <version>${spring.boot.version}</version>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-  </build>
-</project>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
what has been changed here?
The diff shows every line has changed. EOL stlye changed between unix and linux?
The only content change I can see here is that you added `standalone="no"`. What was the reason for that change? I can not see any...